source/deployment/sso-saml.md

```diff
@@ -0,0 +1,47 @@
+## SAML Single-Sign-On Setup (Enterprise) 
+
+### Overview
+
+SAML Single-Sign-On integration offers the following benefits: 
+
+- **Single-sign-on.** Users can sign-in to Mattermost with their SAML credentials.
+- **Centralized identity management.** Mattermost accounts can display user information from SAML, such as first and last name, email and username.
+- **Automatic account provisioning.** New Mattermost user accounts are automatically created the first time a user signs in with their SAML credentials on the Mattermost server. 
+
+Mattermost officially supports Okta and Microsoft ADFS as the identity providers (IDPs), please see links below for more details on how to configure SAML with Okta and Microsoft ADFS. If you'd like, you may also try configuring SAML for a custom IDP.
+
+##### [Configure SAML with Okta](http://docs.mattermost.com/deployment/sso-saml-okta.html) 
+
+##### [Configure SAML with Microsoft ADFS](http://docs.mattermost.com/deployment/sso-saml-adfs.html) 
+
+#### Pre-installation
+
+Before configuring SAML with Okta or Microsoft ADFS, make sure you have the [XML Security Library](https://www.aleksey.com/xmlsec/download.html) installed on your Mattermost instance. The XML Security Library is usually included as part of Debian GNU/Linux.
+
+#### Troubleshooting
```

Comment:
@jasonblais, I'd propose using an include from this section in the other guides instead of duplicating the content. Going to push this in now but we can submit a patch for this onto master.